app/controllers/community/meetups/index.js

```diff
@@ -10,7 +10,7 @@ export default Controller.extend({
   init() {
     this._super(...arguments);
     
-    if (!this.get('fastboot.isFastBoot')) {
+    if (!this.fastboot.isFastBoot) {
```

Comment:
this has actually subtly changed the semantics of the line. Now if fastboot is null or undefined it will throw an error (where it would not have before).

For us to keep the same semantics you would need to do something like this: 

```suggestion
    if (!this.fastboot?.isFastBoot) {
```

Reply:
Updated this.